tests/regression/78-termination/34-nested-for-loop-nonterminating.c

```diff
@@ -3,11 +3,11 @@
 
 int main()
 {
-  int outerCount, innerCount;
+  unsigned int outerCount, innerCount;
 
   for (outerCount = 1; outerCount <= 3; outerCount++)
   {
-    for (innerCount = 1; innerCount > 0; innerCount++) // NONTERMLOOP termination analysis shall mark beginning of for as non-terminating loop
+    for (innerCount = 1; innerCount > 0; innerCount += 4) // NONTERMLOOP termination analysis shall mark beginning of for as non-terminating loop
```

Comment:
What's up with this `+= 4`?

Reply:
Unless the loop condition is modified, this inner loop would terminate once `innerCount` reaches `0`. I thus decided to have a little fun with it, and increment by `4` in each iteration.